src/pages/shared.tsx

```diff
@@ -0,0 +1,74 @@
+import { CardForShared } from 'components/common/CardForShared';
+import SearchBar from 'components/common/SearchBar';
+import ContentLayout from 'components/others/ContentLayout';
+import FolderBanner from 'components/others/FolderBanner';
+import { linkItem } from 'constants/type';
+import { SearchContextProvider } from 'context/SearchContext';
+import { useSearchContext } from 'context/SearchContext';
+import { getSampleUserFolder } from 'pages/api/fetchApi';
+import { useEffect, useState } from 'react';
+import { SampleUserFolder } from 'constants/type';
+import styles from 'styles/folder.module.css';
+
+export default function SharedPage() {
+  return (
+    <SearchContextProvider>
+      <SharedLayout />
+    </SearchContextProvider>
+  );
+}
```

Comment:
[getServerSideProps](https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props)는 역할로 아래처럼 작성하시면 됩니다.

```suggestion
export async function getServerSideProps() {
  const users = await fetch(`http://api.github.com/users`).then(res=> res.json());
  
  return { props: { users } };
}

export default function SharedPage({ users }){
  console.log(users);
  return (
    <SearchContextProvider>
      <SharedLayout />
    </SearchContextProvider>
  );
} 
```

Reply:
직접 작성해주셨군요..참고해서 최대한 바꿔보겠습니다!!